src/env.h

```diff
@@ -1058,6 +1058,9 @@ class Environment : public MemoryRetainer {
   inline bool emit_insecure_umask_warning() const;
   inline void set_emit_insecure_umask_warning(bool on);
 
+  inline void set_source_maps_enabled();
+  inline bool get_source_maps_enabled();
```

Comment:
Idiomatic naming and const correctness:
```suggestion
  inline bool source_maps_enabled() const;
```
The setter should take a `bool`, like the other setters do.